ntbs-service/Pages/Shared/_Header.cshtml

```diff
@@ -56,11 +56,33 @@
                             Reports
                         </a>
                     </li>
-                    <li class="govuk-header__navigation-item header-navigation-link">
-                        <a class="govuk-header__link header-navigation-list-item" href="/Help">
-                            Help
+                    
+                    <navigation-with-submenu inline-template>
+                        <li class="govuk-header__navigation-item header-navigation-link nav-with-submenu-with-js">
+                            <a v-on:click="toggleMenu" class="govuk-header__link header-navigation-list-item nav-with-submenu-header-link">
+                                Help <i class="arrow down"></i>
+                            </a>
+                            <ul ref="navigation-submenu" class="nav-submenu-list" >
+                                <li class="header-navigation-link nav-submenu-list-item">
+                                    <a class="govuk-header__link header-navigation-list-item" href="/Help">
+                                        FAQ
+                                    </a>
+                                </li>
+                                <li class="header-navigation-link nav-submenu-list-item">
+                                    <a class="govuk-header__link header-navigation-list-item" href="/ServiceDirectory">
+                                        Service directory
+                                    </a>
+                                </li>
+                            </ul>
+                        </li>
+                    </navigation-with-submenu>
```

Comment:
Nicely extracted!
Because it's a js-only thing, we can probably make a bit more use of the vue bindings, too, e.g. specifying the conditional rendering directly in the html like:
```
<ul v-if="showMenu" calss=nav-submenu-list">...
```
making use of https://vuejs.org/v2/guide/conditional.html#v-if, meaning we don't have to do the manual `ref` links stuff.
We'll need to add `"showMenu"` to the data held by the component, but I think it'll allow us to get rid of the `ref` stuff, the drilling down by class etc!